x/contractmanager/client/cli/query_failure.go

```diff
@@ -23,19 +26,19 @@ func CmdFailures() *cobra.Command {
 				return err
 			}
 
-			queryClient := types.NewQueryClient(clientCtx)
+			queryClient := contractmanagertypes.NewQueryClient(clientCtx)
 
 			address := ""
 			if len(args) > 0 {
 				address = args[0]
 			}
 
-			params := &types.QueryFailuresRequest{
+			params := &contractmanagertypes.QueryFailuresRequest{
 				Address:    address,
 				Pagination: pageReq,
 			}
 
-			res, err := queryClient.Failures(context.Background(), params)
+			res, err := queryClient.Failures(cmd.Context(), params)
```

Comment:
Shouldn't we pass `clientCtx` here?

Reply:
`clientCtx` is another kind of context as far as I understand. `cmd.Context()` returns the general kind context from golang stdlib whereas `clientCtx` is context in some different terms, and it is not usable as an argument because neither it itself nor one of its fields implement context interface. also it's kind of reasonable to wrap CLI queries into context of cmd execution (`cmd.Context()`) so queries are killed once the parent context (cmd) is killed/interrupted

Reply:
Understandable